presto-docs/src/main/sphinx/release/release-308.rst

```diff
@@ -0,0 +1,56 @@
+===========
+Release 308
+===========
+
+General Changes
+---------------
+
+* Fix a regression that prevented the server from starting on Java 9+. (:issue:`610`)
+* Fix correctness issue for queries involving ``FULL OUTER JOIN`` and ``coalesce``. (:issue:`622`)
+
+Security Changes
+----------------
+
+* Add authorization for listing table columns. (:issue:`507`)
+
+CLI Changes
+-----------
+
+* Add option for specifying Kerberos service principal pattern. (:issue:`597`)
+
+JDBC Driver Changes
+-------------------
+
+* Correctly report precision and column display size in ``ResultSetMetaData``
+  for ``char`` and ``varchar`` columns. (:issue:`615`)
+* Add option for specifying Kerberos service principal pattern. (:issue:`597`)
+
+Hive Connector Changes
+----------------------
+
+* Fix regression that could cause queries to fail with ``Query can potentially
+  read more than X partitions`` error. (:issue:`619`)
+* Improve ORC read performance significantly. For TCP-DS, this saves about 9.5% of total CPU when
```

Comment:
Nit: I would adjust the line breaking so that the two lines are more even in length